v2/car.go

```diff
@@ -1,10 +1,15 @@
 package car
 
+import (
+	"encoding/binary"
+	"io"
+)
+
 const (
 	// HeaderBytesSize is the fixed size of CAR v2 header in number of bytes.
-	HeaderBytesSize = 40
+	HeaderBytesSize uint64 = 40
 	// CharacteristicsBytesSize is the fixed size of Characteristics bitfield within CAR v2 header in number of bytes.
-	CharacteristicsBytesSize = 16
+	CharacteristicsBytesSize uint64 = 16
```

Comment:
any reason to make these typed? afaik there should be none

Reply:
They are used to [calculate offsets](https://github.com/ipld/go-car/blob/b4173dd99357472dd5afb314324db5c96f526425/v2/car.go#L75), the type of which is `uint64`. Making them typed here, and consistently as uint64 avoids the need to cast every time we want to calculate offsets.

Reply:
I'm pretty sure that expression would work with untyped constants. They get converted to whatever type is necessary, implicitly.

Reply:
ah right you are; thank you for pointing this out